management_api_app/service_bus/deployment_status_update.py

```diff
@@ -74,6 +74,14 @@ def create_updated_deployment_document(resource: dict, message: DeploymentStatus
     resource["deployment"]["status"] = message.status
     resource["deployment"]["message"] = message.message
 
+    # although outputs are likely to be relevant when resources are moving to "deployed" status,
+    # lets not limit when we update them and have the resource process make that decision.
+    output_dict = {output.Name: output.Value.strip("'").strip('"') for output in message.outputs}
+
+    params_dict = dict(resource["resourceTemplateParameters"])
```

Comment:
resource["resourceTemplateParameters"] is already a dict afaik and check the code.
one line temp variables are not needed in any case. 
could write `resource["resourceTemplateParameters"].update(output_dict)`